src/appConfig.js

```diff
@@ -31,6 +31,7 @@ if (window.location.host === 'streamstats.usgs.gov') {
         configuration.baseurls.StormRunoffServices = 'https://streamstats.usgs.gov/runoffmodelingservices',
         configuration.baseurls.GageStatsServices = 'https://streamstats.usgs.gov/gagestatsservices',
 		configuration.baseurls.FlowAnywhereRegressionServices = 'https://streamstats.usgs.gov/regressionservices',
+        configuration.baseurls.BatchProcessorServices = 'https://dev.streamstats.usgs.gov/batchprocessor',
```

Comment:
Shouldn't this be ? Or am I confused

```suggestion
        configuration.baseurls.BatchProcessorServices = 'https://streamstats.usgs.gov/batchprocessor',
```

Reply:
Oh yes!